src/tactic/interactive_expr.lean

```diff
@@ -390,8 +391,16 @@ tc.stateless $ λ ft, do
   let lcs := group_local_collection lcs,
   lchs ← lcs.mmap (λ lc, do
     lh ← local_c lc,
+    cls ← is_class lc.type,
     let ns : list (html γ) := lc.locals.map $ λ n,
-      h "span" [cn "goal-hyp b pr2", key n.local_uniq_name] [html.of_name n.local_pp_name],
+      let var_style : string := if ¬ cls
+        then "goal-hyp"
+        else match is with
+        | some is' := if n ∈ is'
+                        then "goal-hyp-inst"
+                        else "goal-hyp-noninst"
+        | none := "goal-hyp-unfrozen"
+        end in h "span" [cn "goal-hyp b pr2", cn var_style] [html.of_name $ expr.local_pp_name n],
```

Comment:
Are you sure this works? I think this might emit `<span class="goal-hyp b pr2" class="$var_style">` which will discard the first list of classnames.

Reply:
@Vierkantor, what's the right way to do this already? I forgot 🤦

Reply:
Probably just `[cn $ "goal-hyp b pr2 " ++ var_style]`?

Reply:
I rather went with `[cn $ var_style ++ " b pr2"]`